core/server/translations/en.json

```diff
@@ -325,7 +325,8 @@
                 "postNotFound": "Post not found."
             },
             "job": {
-                "not-found": "Job not found."
+                "not-found": "Job not found.",
```

Comment:
Just a little cleanup thing here - this key should be camelCase i.e. `notFound` please